ui/src/components/billing/BillingDialogPaymentMethod.vue

```diff
@@ -23,10 +23,19 @@
 
         <v-card-text class="mt-4 mb-3 pb-1">
           <div v-if="typeOperation === 'subscription'">
-            <h4>Subscribe to premium plan</h4>
-            <p>
-              The amount charged will be based on the device usage.
-            </p>
+            <div data-test="subscription-description">
+              <h4>Subscribe to premium plan</h4>
+              <p>
+                The amount charged will be based on the current device usage.
```

Comment:
The subscription is charged monthly, based on the number of devices you have in your namespace.